src/components/Page/Page.tsx

```diff
@@ -32,7 +32,14 @@ export interface Props extends HeaderProps {
   forceRender?: boolean;
 }
 
-export type ComposedProps = Props & WithAppProviderProps;
+export interface DeprecatedProps {
```

Comment:
If we wanted to, we could parse for this interface in the style guide (see bottom of the screenshot)